client/ionic/src/App.tsx

```diff
@@ -33,8 +35,14 @@ const App: React.FC = () => (
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
         <Route path="/menu" component={Menu} exact={true} />
+        <Route
```

Comment:
(nitpick) Use the same code style as the rest of the code please.

Reply:
I tried to do it in one line but it doesn't pass the first two checks on github since it's not formatted the way Prettier wants.

Reply:
We can change the lint config but the idea was to go with an opinionated config.